pkg/registry/core/pod/strategy.go

```diff
@@ -564,3 +533,29 @@ func PortForwardLocation(
 	}
 	return loc, nodeInfo.Transport, nil
 }
+
+// valiateContainer valiate container is valid for pod, return valid container
+func valiateContainer(container string, pod *api.Pod) (string, error) {
+	if container == "" {
```

Comment:
i would recommend to use `len(container)` == 0 like the line `481`

Reply:
> i would recommend to use `len(container)` == 0 like the line `481`

done